htdocs/stripe/class/stripe.class.php

```diff
@@ -352,12 +352,6 @@ public function getPaymentIntent($amount, $currency_code, $tag, $description = '
 
 		if (is_object($object))
 		{
-			// Warning. If a payment was tried and failed, a payment intent was created.
-			// But if we change someting on object to pay (amount or other that does not change the idempotency key), reusing same payment intent is not allowed.
-			// Recommanded solution is to recreate a new payment intent each time we need one (old one will be automatically closed after a delay), Stripe will
-			// automatically return the existing payment intent if idempotency is provided when we try to create the new one.
-			// That's why we can comment the part of code to retreive a payment intent with object id (never mind if we cumulate payment intent with old ones that will not be used)
-			/*
```

Comment:
Can you add a hidden constant condition here
if ($conf->global->STRIPE_REUSE_EXISTING_INTENT_IF_FOUND) {
to enable this code ?
Because i still have trouble of payments no more possible when this part of code is enabled.

Reply:
i have troubles without.... we need more check with client_secret validity in payment form.. i will be done for v12 ;)